docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -76,6 +76,8 @@ These variables allow you to include the current time range in the data link URL
 | `__from`           | For more information, refer to [Global variables][].                |
 | `__to`             | For more information, refer to [Global variables][].                |
 
+When you create data links using time range variables like `__url_time_range` in the URL, you have to form the query parameter syntax yourself; that is, you must add the question mark (`?`) and ampersand (`&`). These characters aren't automatically generated.
```

Comment:
I think there might be a better way of phrasing this - maybe "that is, you must format the url by appending query parameters via the question mark (`?`) and ampersand (`&`) syntax."